template/README.md

```diff
@@ -404,6 +405,44 @@ Note that GitHub Pages doesn't support routers that use the HTML5 `pushState` hi
 
 Use the [Heroku Buildpack for create-react-app](https://github.com/mars/create-react-app-buildpack).
 
+### Adding Custom Environment Variables
+
+> Note: this feature is available with `v0.3.0` and higher.
```

Comment:
Let’s say `react-scripts@0.3.0` to make it clear it’s not the global CLI that needs updating.